decidim-core/app/forms/decidim/account_form.rb

```diff
@@ -17,13 +18,16 @@ class AccountForm < Form
 
     validates :name, presence: true
     validates :email, presence: true
+    validates :nickname, presence: true
 
+    validates :nickname, length: { maximum: Decidim::User.nickname_max_length, allow_blank: true }
```

Comment:
Allow blank? Why?

Reply:
Because there's another presence validation above, check it out! This way the message when there's no nickname set is "required" and not "shorter than expected".

Reply:
Ah cool!